moveit_msgs/RobotTrajectory plan
---
bool success